README.md

```diff
@@ -14,4 +16,48 @@ An inter-agency lens would be applied to determining the selection criteria for
 iii) Pre-arranged finance (the money).
 CERF set aside around $5 million [pending final plan and confirmation] for anticipatory action for floods in
 Bangladesh. This funding will become available immediately once the defined trigger is reached to active the actions described above
-In addition, the pilot seeks to amplify and coordinate similar anticipatory action pilots at the agency scale, including from WFP, IFRC, and others.
\ No newline at end of file
+In addition, the pilot seeks to amplify and coordinate similar anticipatory action pilots at the agency scale, including from WFP, IFRC, and others.
+
+## Analysis
+
+The analysis within this repository contains two components. 
+
+1. Processing FFWC and GLOFAS forecasting data, used to trigger this pilot's anticipatory action. 
+2. Calculating historical estimates of flooding extent over time in five high priority districts (Bogra, Gaibandha, Jamalpur, Kurigram, Sirajganj). This work is largely based on an analysis of Sentinel-1 SAR imagery in Google Earth Engine, accessible [here](https://code.earthengine.google.com/0fe2c1f3b2cf8ef6fe9aa81382b00191). 
+
+## Structure of this repository 
```

Comment:
Really like that you added the structure!

Reply:
Agreed with Tinka's comments. Very well done. WRT the goal: I'll move this conversation to Slack. Overall I think we need to be careful not to over-optimise our code yet: we do a lot of one-off's.